stories/assets/img/welcomebg.svg

```diff
@@ -1,16 +1,16 @@
-<svg width='2880' height='600' xmlns='http://www.w3.org/2000/svg'>
+<svg width='2880' height='1000' xmlns='http://www.w3.org/2000/svg'>
```

Comment:
is there a need for this file to be so large? I noticed a little junk when animations happen, reducing file dimentions could help.